### PR TITLE
test: Add unit tests for `uiRadiobuttons`.

### PR DESCRIPTION
**Pull request by szanni**
Unit tests for uiRadioButtons that I orginally wrote to test #121.

### CODE REVIEW

File: test/unit/radiobuttons.c [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+	uiRadioButtonsAppend(*r, "Item 0");
+	uiRadioButtonsAppend(*r, "Item 1");
+
+	assert_int_equal(uiRadioButtonsSelected(*r), -1);
+}
+
+static void radioButtonsAppendDuplicate(void **state)
+{
+	uiRadioButtons **r = uiRadioButtonsPtrFromState(state);
+	const char *duplciate = "Duplicate";
```
Comment (cody271) [inline test/unit/radiobuttons.c:36]:
```suggestion
	const char *duplicate = "Duplicate";
```

File: test/unit/radiobuttons.c [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+	*r = uiNewRadioButtons();
+
+	uiRadioButtonsAppend(*r, duplciate);
```
Comment (cody271) [inline test/unit/radiobuttons.c:40]:
```suggestion
	uiRadioButtonsAppend(*r, duplicate);
```

File: test/unit/radiobuttons.c [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+	uiRadioButtonsAppend(*r, duplciate);
```
Comment (cody271) [inline test/unit/radiobuttons.c:41]:
```suggestion
	uiRadioButtonsAppend(*r, duplicate);
```

**Review by cody271 — COMMENTED** [review]
Nits

File: test/unit/radiobuttons.c (hunk 1 above)
Comment (szanni) [reply test/unit/radiobuttons.c]: Good catch. Fixed

**Review by cody271 — APPROVED** [review]
LGTM

**Pull request merged**